### PR TITLE
docs: initial draft

### LINKED ISSUE #2: [Paper]: DINO
### Title of the Paper

Emerging Properties in Self-Supervised Vision Transformers

### What Year was the paper published in?

2021

### Link to the paper

https://arxiv.org/abs/2104.14294

### PR DESCRIPTION
**Pull request by SauravMaheshkar**
Initial Draft of the List.

Also fixes #2

### CODE REVIEW

File: .github/workflows/links.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: "Check Links"
+
+on:
+    push:
+      branches:
+        - master
```
Comment (guarin) [inline .github/workflows/links.yml]:
```suggestion
        - main
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,79 @@
-# Awesome Self Supervised Learning [![awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
+# Awesome Self Supervised Learning [![awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome) [![Discord](https://img.shields.io/discord/752876370337726585?logo=discord&logoColor=white&label=discord&color=7289da)](https://discord.gg/xvNJW94)
```
Comment (guarin) [inline README.md:1]: Also add links to:
* LightlySSL github repo
* Lightly website

In a future PR we should also add links to other popular awesome ssl lists.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,79 @@
...
 
+## 2024
+
+| Title | Relevant Links |
+|:-----:|:--------------:|
```
Comment (guarin) [inline README.md]: Let's left-align everything, it makes screening the list much easier.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,79 @@
...
+| [Scalable Pre-training of Large Autoregressive Image Models](https://arxiv.org/abs/2401.08541) | [![arXiv](https://img.shields.io/badge/arXiv-2401.08541-b31b1b.svg)](https://arxiv.org/abs/2401.08541) [![Open In Colab](https://img.shields.io/badge/Colab-PyTorch-blue?logo=googlecolab)](https://colab.research.google.com/github/lightly-ai/lightly/blob/master/examples/notebooks/pytorch/aim.ipynb) |
```
Comment (guarin) [inline README.md:9]: Awesome that you used the arxiv batch!

**Review by guarin — CHANGES_REQUESTED** [review]
This looks awesome! Left some minor comments.

File: .github/workflows/links.yml [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    pull_request:
+    workflow_dispatch:
+
+jobs:
+  check-links:
```
Comment (guarin) [inline .github/workflows/links.yml:11]: Did you check that this fails if a link is broken?

File: README.md [hunk 6]
```diff
@@ -1,2 +1,81 @@
-# Awesome Self Supervised Learning [![awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
+# Awesome Self Supervised Learning [![awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome) [![Discord](https://img.shields.io/discord/752876370337726585?logo=discord&logoColor=white&label=discord&color=7289da)](https://discord.gg/xvNJW94)
 
+Checkout [Lightly**SSL**](https://github.com/lightly-ai/lightly) a computer vision framework for self-supervised learning by the team at [lightly.ai](https://www.lightly.ai/).
+
+## 2024
+
+| Title | Relevant Links |
+|:-----|:--------------|
+| [Scalable Pre-training of Large Autoregressive Image Models](https://www.google.com/404) | [![arXiv](https://img.shields.io/badge/arXiv-2401.08541-b31b1b.svg)](https://arxiv.org/abs/2401.08541) [![Open In Colab](https://img.shields.io/badge/Colab-PyTorch-blue?logo=googlecolab)](https://colab.research.google.com/github/lightly-ai/lightly/blob/master/examples/notebooks/pytorch/aim.ipynb) |
```
[the remark this answers is not in the document]
Comment (SauravMaheshkar) [reply README.md]: @guarin the CI does indeed fail for a broken link

**Review by guarin — APPROVED** [review]
LGTM!

**Pull request merged**